src/options/output.rs

```diff
@@ -275,6 +275,6 @@ pub enum JsonVersion {
     #[serde(rename = "1")]
     #[default]
     One,
-    #[serde(rename = "2")]
-    Two,
+    #[serde(rename = "2.1")]
+    TwoOne,
```

Comment:
This should just go right to `3` I think because it's a fully breaking change as you can no longer access via `.data.tasks.lint` and would have to update scripts to `.data.tasks.linter`.

Reply:
Resolving this comment since we agreed to not change the JSON output as part of this PR. 

Reverted those changes as part of [8940fe7](https://github.com/apollographql/rover/pull/1701/commits/8940fe7c4bf88472a101dfa23d2fa052ac1c9934).